packages/rspack-test-tools/tests/configCases/builtins/provide/index.js

```diff
@@ -9,13 +9,37 @@ it("should provide a module for a nested var", function () {
 });
 
 it("should provide a module for a nested var within a IIFE's argument", function () {
-	(function (process) {
-		expect(process.env.NODE_ENV).toBe("production");
```

Comment:
The original case is wrong and it's wrongly replaced by `DefinePlugin` to `"production"`. However, it should leverage `ProvidePlugin`.